ee/settings.py

```diff
@@ -26,5 +26,9 @@
     ].split(",")
 
 # ClickHouse and Kafka
-CLICKHOUSE_DENORMALIZED_PROPERTIES = os.getenv("CLICKHOUSE_DENORMALIZED_PROPERTIES", "").split(",")
+CLICKHOUSE_DENORMALIZED_PROPERTIES: list = (
+    os.getenv("CLICKHOUSE_DENORMALIZED_PROPERTIES", "").split(",")
```

Comment:
mypy woes for the default arg. We know if won't be empty because of the next line, but oh well.